apps/api/src/domain/models/tag/tag.entity.ts

```diff
@@ -226,4 +236,9 @@ export class Tag extends Aggregate implements HasLabel {
 
         return buildResult;
     }
+
+    // TODO put this on the base aggregate root class
```

Comment:
Oh we can probably do that when we rebase by following the `BaseEvent` implementation. It will be a little different because the aggregate root Ctors have different signatures, though.